conf.d/postgres.yaml.example

```diff
@@ -6,6 +6,7 @@ instances:
 #    username: my_username
 #    password: my_password
 #    dbname: db_name
+#    ssl: True_or_False
```

Comment:
In most of the other checks configs, we just put the default value (for a bool it's pretty self-explanatory)
```
# ssl: false
```